docs2/site/docs/getting-started/schema-types.md

```diff
@@ -276,3 +276,66 @@ be implemented as:
         }
     }
 ```
+
+## Type Mapping
```

Comment:
See `Ability to map CLR types to GraphTypes` section in v4 migration guide. It's worth to mention `GraphQLMetadata` attribute here as an alternative way to define the mapping.

Reply:
You're right, it should be mentioned here.  Ugh I thought I was done.  Well, maybe I can get to it tomorrow night.

Reply:
Done